polyphemus/lib/etls/ipi/ipi_rna_seq_add_processed_files_watch_folders_etl.rb

```diff
@@ -6,8 +6,8 @@ class Polyphemus::IpiRnaSeqAddProcessedFilesWatchFoldersEtl < Polyphemus::AddWat
   BUCKET = "data"
 
   def initialize
-    @linker = Polyphemus::IpiRnaSeqProcessedFilesLinker.new
     super(
+      linker: Polyphemus::IpiRnaSeqProcessedFilesLinker.new(project_name: PROJECT, bucket_name: BUCKET),
```

Comment:
More for single-cell than bulk RNA, perhaps, but if we ever instantiate an ETL with multiple project / bucket pairs, will this break since it's coupled to a single project / bucket?

Reply:
Yes.  But, I intend to break this apart in my follow up PR.  This is functionally equivalent to its original form other than passing through the constructor, but I intend to separate the ETL and linker out a bit more with some additional refactoring.  I didn't want to crowd up this PR too much.

Reply:
Ah, you ean, that the the project and bucket come from the constructor and _not_ the link execution?
Good point.  I may revert that.

Reply:
I think my original motivation was the simplicity of helper methods not having to pass these values around.  I think I will keep the linker having these values in the constructor, and simply say that each processing creates a new linker relative to its active cursor value.